tests/main/views/test_suppliers.py

```diff
@@ -1511,7 +1516,38 @@ def supplier_framework_interest(self, supplier_framework, update):
                 }),
             content_type='application/json')
 
-    def test_adding_supplier_has_passed(self, supplier_framework):
+    @staticmethod
+    def _refetch_serialized_sf(supplier_framework):
+        # must be performed within an app context
+        return SupplierFramework.query.filter(
+            SupplierFramework.framework.has(Framework.slug == supplier_framework["frameworkSlug"])
+        ).filter(
+            SupplierFramework.supplier_id == supplier_framework["supplierId"]
+        ).order_by(Supplier.id.asc()).first().serialize()
+
+    @staticmethod
+    def _latest_supplier_update_audit_event(supplier_id):
+        # must be performed within an app context
+        return AuditEvent.query.filter(
+            AuditEvent.object == Supplier.query.filter(
+                Supplier.supplier_id == supplier_id
+            ).first(),
+            AuditEvent.type == "supplier_update",
+        ).order_by(AuditEvent.created_at.desc()).first()
+
+    @classmethod
+    def _assert_and_return_audit_event(cls, supplier_framework):
+        # must be performed within an app context
+        audit = cls._latest_supplier_update_audit_event(supplier_framework['supplierId'])
+        assert audit.type == "supplier_update"
+        assert audit.user == "interested@example.com"
```

Comment:
I hadn't really intended for this to be used outside `TestSupplierFrameworkUpdates` - if someone finds it useful they can yank it out when they transfer it I guess?